### ISSUE TITLE
[Doc] Improvement to "Switching Release Channels"

### ISSUE THREAD

**Issue by rosskukulinski** [coreos/docs#426]
From [this](https://github.com/coreos/docs/blob/master/cluster-management/setup/switching-channels/index.md)

I think it's important to put a disclaimer that your cloud_config will (probably?) override any changes you make to release channel OR reboot strategy the next time the system boots.

I'm happy to open PR to add an appropriately worded message, but I'd like someone more knowledgeable to sanity check my above assertion.

**Comment by robszumski** [comment]
@crawford Can you verify that these situations are correct and then we can figure out how best to document:

## Manual Change

Machine is booted on stable channel. After boot, `/etc/coreos/update.conf` is created and specifies beta.
**Now**: machine tracks stable
**After update-engine restart**: machine tracks beta
**Next Reboot**: machine tracks beta

## Cloud-config with manual change

Machine is booted on stable channel with cloud-config specifying alpha. After boot, `/etc/coreos/update.conf` is created and specifies beta.
**Now**: machine tracks alpha
**After update-engine restart**: machine tracks beta
**Next Reboot**: machine tracks alpha

*if true, this is misleading/confusing*

## Cloud-config only

Machine is booted on stable channel with cloud-config specifying beta.
**Now**: machine tracks beta
**Next Reboot**: machine tracks beta

**Comment by rosskukulinski** [comment]
thanks @robszumski - that tracks how I have found update.conf to behave.

**Comment by crawford** [comment]
This is correct. In fact, we go out of our way to only overwrite the fields in `/etc/coreos/update.conf` that differ from the cloud-config. This feels incorrect. I don't think we should touch that file.

**Comment by robszumski** [comment]
@crawford It sounds like this would be better if it matched the drop-in workflow we have for other services. I know thats probably a bigger change to update-engine than we want to make right now.

**Comment by crawford** [comment]
Yes and yes. For starters, we should fix cloudinit.
